pkgs/development/python-modules/pick-colour-picker/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, pygobject3
+, pycairo
+, glib
+, gtk3
+, gobject-introspection
+, wrapGAppsHook
+}:
+
+buildPythonPackage rec {
+  pname = "pick-colour-picker";
+  version = "1.5.0-3ec940";
+
+  src = fetchFromGitHub {
+    owner = "stuartlangridge";
+    repo = "ColourPicker";
+    rev = "3ec9406d787ce373f6db0d520ed38a921edb9473";
+    sha256 = "04l8ch9297nhkgcmyhsbg0il424c8vy0isns1c7aypn0zp0dc4zd";
+  };
+
+  propagatedBuildInputs = [
```

Comment:
Regular dependencies should go to `buildInputs` and Python libraries to `propagatedBuildInputs` or better `pythonPath` (it appears it is no longer deprecated).